woocommerce/woocommerce
# Extend cart checkout block package: Prep for publish

### All Submissions:






### Changes proposed in this Pull Request:

In order to publish `@woocommerce/extend-cart-checkout-block`, a `.gitignore` file is needed to prevent the published package from containing development files such as `vendor/automattic/jetpack-changelogger`. Previously, there was a `.gitignore.mustache` file. I have no experience with Mustache, so I'm not certain why that is but this seems like a proper change to make for the sake of IDE's.

### How to test the changes in this Pull Request:

1. `cd packages/js/extend-cart-checkout-block`
2. Perform a publish dry run
```
pnpm publish --publish-branch try/publish-extend-cart-checkout-block --dry-run
```
3. See the output list the correct file to include in the published package

### Other information:




### FOR PR REVIEWER ONLY:
